test/rules/mappings/test_used.py

```diff
@@ -0,0 +1,56 @@
+"""
+  Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+  Permission is hereby granted, free of charge, to any person obtaining a copy of this
+  software and associated documentation files (the "Software"), to deal in the Software
+  without restriction, including without limitation the rights to use, copy, modify,
+  merge, publish, distribute, sublicense, and/or sell copies of the Software, and to
+  permit persons to whom the Software is furnished to do so.
+
+  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+  INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+  PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+  HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+  OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+  SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+"""
+from cfnlint.rules.mappings.Used import Used  # pylint: disable=E0401
+from .. import BaseRuleTestCase
+
+
+class TestUsedMappings(BaseRuleTestCase):
+    """Test template mapping configurations"""
+    def setUp(self):
+        """Setup"""
+        super(TestUsedMappings, self).setUp()
+        self.collection.register(Used())
+
+    success_templates = [
```

Comment:
What happens if you just run this on the base's success_templates?

This override was added to the Conditions version because too much of the "good" quickstart templates became invalid (see comments in the PR of the Conditions Used rule)

Reply:
There are a couple of QuickStarts with unused Mappings, so I had to use the same trick.  @kddejong and I discussed moving this to an "exclude" model to make it easier to maintain going forward (especially if we add more sample templates).

I'll create a new issue recommending that approach going forward.